src/main/scala/beam/router/BeamRouter.scala

```diff
@@ -176,6 +163,11 @@ class BeamRouter(
       }
     case GetMatSimNetwork =>
       sender ! MATSimNetwork(network)
+    case GetTravelTime =>
```

Comment:
Why move this from R5Routing to here?

Reply:
I've moved it out because I don't want us to send this data across the network. BeamRouter sits clother to simulation (the same JVM), so if it will be sent there, it's just a reference.